src/dev/eslint/types.eslint.config.template.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
   },
   overrides: [
     {
-      files: ['server/**/*'],
+      files: ['server/**/*', '*functional*/**/*', '*api_integration*/**/*'],
```

Comment:
I think it is a good start, PR has quite many changes. We have more tests, but I think it is better to handle separately